geopandas/array.py

```diff
@@ -1486,7 +1486,8 @@ def _get_common_crs(arr_seq):
             warnings.warn(
                 "CRS not set for some of the concatenation inputs. "
                 f"Setting output's CRS as {names[0]} "
-                "(the single non-null crs provided)."
+                "(the single non-null crs provided).",
+                stacklevel=1,
```

Comment:
```suggestion
                stacklevel=2
```

Those `stacklevel=1` cases are AFAIK always wrong, the minimum to use should be 2. Although in this case it might be difficult to exactly determine the appropriate stacklevel, as it will depend on the exact code path in pandas to get here (and there can be multiple)

(unless the stacklevel=1 is done deliberately to point at the code here, in case we can't point to the user's calling code, to avoid pointing somewhere random in the call stack)

Reply:
I used 1 here specifically because the correct level is variable but we can as well use 2.